.eslintrc

```diff
@@ -1,7 +1,12 @@
-extends: universal-search
+extends: universal-search/addon
+
+ecmaFeatures:
+  generators: true
 
 rules:
-  consistent-return: 0
+  func-names: 0
   no-console: 0
-  no-var: 2
-  prefer-const: 2
+  no-nested-ternary: 1
+  no-param-reassign: 1
+  object-shorthand: 0
+  space-before-function-paren: [2, never]
```

Comment:
I think airbnb's style guide now prefers spaces before function parens, but If I enable (or remove) this rule, I get ~65 errors. So it was easier for me to remove the extraneous spacing in 7 places, rather than tackle the 65 other places. Although I'll be happy to normalize that in a future PR if you want us to re-align with airbnb's rules.